main.cpp

```diff
@@ -92,6 +92,91 @@ good to go!
  */
 
 #include <iostream>
+
+struct FloatType
+{
+    FloatType() {}
+
+    float add(float lhs, float rhs)
+    {
+        return lhs + rhs;
+    }
+
+    float subtract(float lhs, float rhs)
+    {
+        return lhs - rhs;
+    }
+
+    float multiply(float lhs, float rhs)
+    {
+        return lhs * rhs;
+    }
+
+    float divide(float lhs, float rhs)
+    {
+        if(rhs == 0.0f) std::cout << "\nwarning, floating point division by zero returns 'inf' !\n";
+        return lhs / rhs;
+    }
```

Comment:
put single-line if() blocks on their own line
This makes the code more readable and logic easier to understand
- [ ] revise